docs/tutorials/clusters/multiple-schedulers.md

```diff
@@ -89,6 +89,41 @@ my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
 You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
 pod in this list.
 
+To run multiple-scheduler with leader election enabled, you must do the following:
+
+First, update the following fields in your YAML file:
+```
+ --leader-elect=true
+ --lock-object-namespace=lock-object-namespace
+ --lock-object-name=lock-object-name
+```
+
+If RBAC is enabled on your cluster, you must update the system:kube-scheduler cluster role. Add you scheduler name to the resourceNames of the rule applied for endpoints resources, as in the following example:
```

Comment:
Please use back-ticks around "system:kube-scheduler" so that it uses the code highlight format, i.e.:

\`system:kube-scheduler\` => `system:kube-scheduler`